### PR TITLE
Add a users table for developers

### PR DESCRIPTION
**Pull request by ctvidic**
Ticket #180678039

- Adds table migration/model for developer users accommodating slack handle, github handle, and public key

### CODE REVIEW

File: app/models/developer_user.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class DeveloperUser < ApplicationRecord
+    validates :public_key, presence: true
```
Comment (aaddaamm) [inline app/models/developer_user.rb:2]: CI is whining about using consistent spacing for this file, just back this spacing off to 2 spaces and it should be good.

Comment (mariana0pachon) [reply app/models/developer_user.rb:2]: ☝️ also, might be good to change the default on your editor

**Review by rlamsal1256 — APPROVED** [review]
Nice work 👍

**Review by cbarber — APPROVED** [review]
LGTM :rocket:

**Pull request merged**